Automattic/jetpack
# Admin Page: Move to Core translation functions -- PART 4

#### Changes proposed in this Pull Request:

This is part 4, follow-up of #13527. We're moving away from `translate` and using the `@wordpress/i18n` package instead.

This PR focusses on updating 3 more parts of the dashboard:
- The Security settings
- The Privacy settings (see link in the footer)
- The search functionality

**Note: this PR is open against the branch used for part 1 in #13527 for now, hence the "Do not merge" label. Once the other PR is merged I'll update the base branch for this PR.**

#### Jetpack product discussion

* p1HpG7-99c-p2
* Primary issue: #16481 

#### Does this pull request change what data or activity we track or use?

* N/A

#### Testing instructions:

Nothing should change at first sight.

* Check the modified pages for any changes; you shouldn't see anything odd.

#### Proposed changelog entry for your changes:

* TBD
